src/main/java/com/android/volley/cronet/CronetHttpStack.java

```diff
@@ -0,0 +1,293 @@
+package com.android.volley.cronet;
```

Comment:
just realized I forgot about this for previous new files, but all Java files should have a copyright header at the top (matching existing ones, with the year updated).

for this PR, just handle the new files in this class, but could you also send a follow-up adding it for the new files you've added in previous PRs?

Reply:
oh yeah, I'll add those to all the new files.